r2r/examples/quickstart.py

```diff
@@ -50,9 +50,9 @@ def __init__(
             config_name = "default"
```

Comment:
So, this is not the way to go around setting a default for R2R via the config.

You will need to go deeper into the app logic, because for the docker image for instance we do not even use the quickstart server. You will need to use the config to override default behavior on the retrieval / ingestion / .. services, basically anywhere .generate is being called we should be injecting the config.